src/Bundle/Controller/ControllerTrait.php

```diff
@@ -156,13 +158,19 @@ protected function file($file, string $fileName = null, string $disposition = Re
      *
      * @final
      */
-    protected function addFlash(string $type, $message)
+    protected function addFlash(string $type, mixed $message): void
```

Comment:
Hum, is this change necessary? I think it's a bc-break cause the method is protected.

Reply:
I have nothing else to add. Other then that, it looks good 👍